dockerfiles/ruby.dockerfile

```diff
@@ -22,6 +22,7 @@ RUN \
   rm -rf /var/lib/apt/lists/*
 
 ARG RAILS_VERSION
+RUN gem update --system 3.3.26
```

Comment:
Could you add a comment what you described in the PR description here too?
It'll be easy to refer on maintenance.

Reply:
Added the comment with a slight update on the code.